kong/runloop/handler.lua

```diff
@@ -623,8 +626,9 @@ end
 do
   local router
   local router_version
-  local router_cache = lrucache.new(Router.MATCH_LRUCACHE_SIZE)
-  local router_cache_neg = lrucache.new(Router.MATCH_LRUCACHE_SIZE)
+  local router_cache_size = Router.DEFAULT_MATCH_LRUCACHE_SIZE
```

Comment:
Should we add a line `local DEFAULT_MATCH_LRUCACHE_SIZE = Router.DEFAULT_MATCH_LRUCACHE_SIZE`

Reply:
@chronolaw Could you make the change and I will merge it.